webapp/templates/jury/problem_testcases.html.twig

```diff
@@ -94,7 +94,7 @@
                 <tr>
                     <td class="filename">
                         <a href="{{ path('jury_problem_testcase_fetch', {'probId': problem.probid, 'rank': testcase.rank, 'type': 'output'}) }}">
-                            {{ testcase.downloadName }}.out
+                            {{ testcase.downloadName }}.ans
```

Comment:
Here we don't use the class constant anymore?

Reply:
I can change that. Maybe set it in the controller as var since constants I’m twig look kinda ugly

Reply:
Makes sense to me because having one other place where it's hardcoded that will definitely be overlooked

Reply:
Agreed. Will do it